include/DhyanaCamera.h

```diff
@@ -135,6 +169,14 @@ class LIBDHYANA_API Camera
     void getFirmwareVersion(std::string& version);
     bool isAcqRunning() const;
 
+    void getFPS(double& fps);	
+    void getTriggerMode(TucamTriggerMode& mode){mode = m_tucam_trigger_mode;};
+    void setTriggerMode(TucamTriggerMode mode){m_tucam_trigger_mode = mode;};
+    void getTriggerEdge(TucamTriggerEdge& edge){edge = m_tucam_trigger_edge_mode;};
+    void setTriggerEdge(TucamTriggerEdge edge){m_tucam_trigger_edge_mode = edge;};
+    void getOutputSignal(int port, TucamSignal& signal, TucamSignalEdge& edge, int& delay, int& width);
+    void setOutputSignal(int port, TucamSignal signal, TucamSignalEdge edge=SignalEdgeRising, int delay=-1, int width=-1);
+
```

Comment:
move the functions impl into the .cpp , like for the getFPS()
also add  DEB_MEMBER_FUNCT()